artifactory_cleanup/rules/base.py

```diff
@@ -91,8 +91,10 @@ def name(cls) -> str:
     @classmethod
     def title(cls) -> str:
         """Cut the docstring to show only the very first important line"""
-        docs = [x.strip() for x in cls.__doc__.splitlines() if x][0]
-        return docs
+        if cls.__doc__:
+            return [x.strip() for x in cls.__doc__.splitlines() if x][0]
+        else:
+            return ""
```

Comment:
Let's handle the edge case first to highlight the main flow

```suggestion
        if not cls.__doc__:
            return ""
        return [x.strip() for x in cls.__doc__.splitlines() if x][0]
```